core/app/c/[communitySlug]/forms/[formSlug]/edit/EditFormTitleButton.tsx

```diff
@@ -0,0 +1,117 @@
+"use client";
+
+import React, { useState } from "react";
+import { zodResolver } from "@hookform/resolvers/zod";
+import { useForm } from "react-hook-form";
+import { z } from "zod";
+
+import { Button } from "ui/button";
+import { Dialog, DialogContent, DialogFooter, DialogTitle, DialogTrigger } from "ui/dialog";
+import { Form, FormControl, FormField, FormItem, FormLabel, FormMessage } from "ui/form";
+import { Check, X } from "ui/icon";
+import { Input } from "ui/input";
+import { toast } from "ui/use-toast";
+
+import { didSucceed, useServerAction } from "~/lib/serverActions";
+import { updateForm } from "./actions";
+
+const editFormTitleSchema = z.object({
+	name: z.string().min(1, "Form name is required"),
+});
+
+type NotificationProps = {
+	message: string;
+	success?: boolean;
+};
+
+const NotificationComponent: React.FC<NotificationProps> = ({ message, success = true }) => {
+	return (
+		<div
+			className={`mx-auto flex items-center ${
+				success
+					? "border-green-200 bg-green-100 text-green-800"
+					: "border-red-200 bg-red-100 text-red-800"
+			}`}
+		>
+			<div className="mr-3 text-2xl">{success ? <Check /> : <X />}</div>
+			<div className="text-lg font-semibold">{message}</div>
+		</div>
+	);
+};
```

Comment:
https://github.com/pubpub/v7/blob/b94ac8bbbdd4533305cc08e8affdf13ff35f9ac9/core/app/components/FormBuilder/FormBuilder.tsx#L178-L185

Could you use this implementation instead? The point of the component was to make sure these two toasts are the same, and matching the design:
![image](https://github.com/user-attachments/assets/4d797a95-189c-4bf8-aa4a-bb3fd5dbe4da)

Current save toast:
![image](https://github.com/user-attachments/assets/78c72c43-d9da-46fd-9bc8-fb5c709518f3)

Toast on this branch:
![image](https://github.com/user-attachments/assets/e071aef0-8e90-4247-a80f-e89a081c2f44)